src/faction_camp.cpp

```diff
@@ -3126,24 +3018,24 @@ int talk_function::camp_recruit_evaluation( const std::string &base,
     }
     return sbase + sexpansions + sfaction + sbonus;
 }
-int talk_function::camp_recruit_evaluation( const std::string &base,
-        const std::vector<std::pair<std::string, tripoint>> &om_expansions )
+int basecamp::recruit_evaluation() const
 {
     int sbase;
     int sexpansions;
     int sfaction;
     int sbonus;
-    return camp_recruit_evaluation( base, om_expansions, sbase, sexpansions, sfaction, sbonus );
+    return recruit_evaluation( sbase, sexpansions, sfaction, sbonus );
 }
 
-std::string talk_function::camp_recruit_start( npc &p, const std::string &base,
-        const std::vector<std::pair<std::string, tripoint>> &om_expansions )
+std::string talk_function::camp_recruit_start( npc &p )
 {
+    const tripoint omt_pos = p.global_omt_location();
+    basecamp *bcp = g->m.camp_at( omt_pos );
```

Comment:
This uses the wrong coordinate system `map::camp_at` expects a local map square (same as for `map::has_furn`for example), but `glbal_omt_location` returns a global overmap terrain coordinate.

Reply:
Oops.  Copy-pasta error.  Fixed.